veps/sig-compute/decentralized-migration/decentralized-migration-vep.md

```diff
@@ -0,0 +1,279 @@
+# VEP #Decentralized live migration
+
+## Release Signoff Checklist
+
+Items marked with (R) are required *prior to targeting to a milestone / release*.
+
+- [ ] (R) Enhancement issue created, which links to VEP dir in [kubevirt/enhancements] (not the initial VEP PR)
+
+## Overview
+
+Proposal to decentralize live migration into several flows to allow for more flexible live migration. This includes live migration to a different VM in a different namespace or cluster.
+
+## Motivation
+
+Current live migration is limited in its flexiblity because the migration is centralized. This limits live migration to the same namespace and cluster. There are use cases where it would be beneficial to live migrate between namespaces and even between clusters.
+
+In order to support those use cases, it is needed that live migration is decentralizated into separate flows:
+
+- synchronization
+- source
+- target
+
+The synchronization flow establishes a connection between the source and target and synchronizes the `VirtualMachineInstance` status between the two. 
```

Comment:
Why do we want kubevirt to estbalish the connection.
Could it help to simply allow kubevirt to send and receive a stream fro msomewhere? This would decouple the responsibilities.

Reply:
I am not quite sure what you mean here. The migration happens deep in the bowels of kubevirt. And thus it should be the one creating the connections as it has all the needed information to do so. You want some outside entity creating these connections and somehow passing them to kubevirt? I am not sure I follow how that would be possible.